src/main/java/com/treasure/hunt/view/StatisticTableController.java

```diff
@@ -232,4 +251,144 @@ public void onSeriesRun(ActionEvent actionEvent) {
+        table.get(0).addAll(instanceStatisticsTableView.getItems()
+                .stream()
+                .map(statisticsWithId -> Integer.toString(statisticsWithId.getId()))
+                .collect(Collectors.toList()));
+
+        for (StatisticObject.StatisticInfo statisticInfo : statisticsMeasureHashMap.keySet()) {
+            List<String> row = new ArrayList<>();
+            row.add(statisticInfo.getName());
+            row.addAll(statisticsMeasureHashMap.get(statisticInfo)
+                    .stream()
+                    .map(statisticObject -> statisticObject.getValue().toString())
+                    .collect(Collectors.toList()));
+            table.add(row);
+        }
+        table = ListUtils.transpose(table);
+        writer.writeAll(table
+                .stream()
+                .map(strings -> strings.toArray(String[]::new))
+                .collect(Collectors.toList()));
+        writer.close();
+    }
+
+    public void onPlot() {
+
+        Stage stage = new Stage();
+        stage.setTitle("Statistic Plot");
+
+        final NumberAxis xAxis = new NumberAxis();
+        final NumberAxis yAxis = new NumberAxis();
+
+        final LineChart<Number, Number> lineChart =
+                new LineChart<Number, Number>(xAxis, yAxis);
+
+        XYChart.Series series = new XYChart.Series();
+        Class<? extends GameEngine> selectedGameEngine = gameEngineList.getSelectionModel().getSelectedItem();
+        Class<? extends Searcher> selectedSearcher = searcherList.getSelectionModel().getSelectedItem();
+        Class<? extends Hider> selectedHider = hiderList.getSelectionModel().getSelectedItem();
+
+        Septet<String, String, Integer, Integer, Integer, Double, Boolean> settings = JavaFxUtils.getSettingsViaDialog(
+                Arrays.stream(selectedHider.getFields())
+                        .filter(field -> field.getType() == String.class)
+                        .map(Field::getName)
+                        .collect(Collectors.toList())
+        );
+
+        if (settings == null) {
+            return;
+        }
+
+        int upperBound = settings.getValue2();
+        int lowerBound = settings.getValue3();
+        Double stepSize = settings.getValue5();
+        boolean png = settings.getValue6();
+
+        new Thread(new Runnable() {
+            List<Double> yValues = new ArrayList<>();
+
+            public void run() {
+                try {
+                    yValues = Statistic.computeMultipleSeries(settings, selectedGameEngine, selectedSearcher, selectedHider);
+                } catch (InvocationTargetException e) {
+                    e.printStackTrace();
+                } catch (NoSuchMethodException e) {
+                    e.printStackTrace();
+                } catch (InstantiationException e) {
+                    e.printStackTrace();
+                } catch (IllegalAccessException e) {
+                    e.printStackTrace();
+                }
```

Comment:
dont use printStackTrace and if u do allways the same for every Exception why dont u catch Exception then, also probably rethrow or return because code blow is useless will throw an Exception as well if yValues is not set